project/src/RandomUtils.gd

```diff
@@ -0,0 +1,15 @@
+extends Object
```

Comment:
What do you think about having a "Utils" class, with inner classes.
As an example:
```
extends Object
classname Utils

class rand
    var num
    func generate_rand_binary():
        ...

class tree
    var num
    func binary_search():
        ...

...
```

Godot has some documentation on this usage: https://docs.godotengine.org/en/3.1/getting_started/scripting/gdscript/gdscript_basics.html?highlight=class#inner-classes

Let me know what you think!

Reply:
Done.